docs/sources/flow/reference/components/otelcol.processor.filter.md

```diff
@@ -0,0 +1,302 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.processor.filter/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.processor.filter/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.processor.filter/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.filter/
+labels:
+  stage: experimental
+title: otelcol.processor.filter
+description: Learn about otelcol.processor.filter
+---
+
+# otelcol.processor.filter
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" version="<AGENT VERSION>" >}}
+
+`otelcol.processor.filter` accepts and filters telemetry data from other `otelcol`
+components using the [OpenTelemetry Transformation Language (OTTL)][OTTL].
+OTTL statements consist of [OTTL Converter functions][], which act on paths.
+A path is a reference to a telemetry data such as:
+* Resource attributes.
+* Instrumentation scope name.
+* Span attributes.
+If any condition is met, the telemetry data is dropped (each condition is ORed together)
+
+In addition to the [standard OTTL Converter functions][OTTL Converter functions], 
+there is also a set of metrics-only functions exclusively by the processor
```

Comment:
```suggestion
In addition to the [standard OTTL Converter functions][OTTL Converter functions], 
the following metrics-only functions are used exclusively by the processor:
```

I'm not sure this rephrase is accurate. The original looks like it's missing some words... and this is my best guess at what was intended.